### PR TITLE
docker instructions added

### LINKED ISSUE #5: Add Installation instructions using Docker
We have setup the docker for ngx but  the docs are still missing. The deliverable of this issue is to add the installation instructions using docker. Please feel free to comment if you have concerns. 

Command to run docker: `docker-compose -f docker-compose.dev.yml up -d --build`

### PR DESCRIPTION
**Pull request by muddlebee**
fix #5 @deshraj @RishabhJain2018 please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.
 
 Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
 
-## Further help
+## Docker Setup
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.6.5.
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+git clone https://github.com/Cloud-CV/EvalAI-ngx.git && cd EvalAI-ngx
```
Comment (aka-jain) [inline README.md]: @anweshknayak please refer steps from here https://github.com/Cloud-CV/EvalAI/blob/master/README.md#using-docker
Write down in this way.

**Comment by muddlebee** [comment]
@aka-jain done. Also should I add this
 `cp settings/dev.sample.py settings/dev.py` ? I have skipped this part for now

File: README.md [hunk 2]
```diff
@@ -28,8 +28,18 @@ Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.
 ## Running end-to-end tests
 
 Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
+### Using Docker
```
Comment (deshraj) [inline README.md]: Change it to `## Setup using Docker`

**Review by deshraj — CHANGES_REQUESTED** [review]

**Comment by muddlebee** [comment]
@deshraj updated

**Review by deshraj — APPROVED** [review]

**Pull request merged**